grails-app/views/wps/_asyncRequest.xml.gsp

```diff
@@ -9,31 +9,30 @@
     <ows:Identifier>gs:Notifier</ows:Identifier>
     <wps:DataInputs>
         <wps:Input>
-            <ows:Identifier>notifiable</ows:Identifier>
+            <ows:Identifier>wrappedProcessResponse</ows:Identifier>
             <wps:Reference mimeType="application/octet-stream" xlink:href="http://geoserver/wps" method="POST">
                 <wps:Body>
                     <wps:Execute version="1.0.0" service="WPS">
                         <ows:Identifier>gs:${jobType}</ows:Identifier>
-
+                        <ows:Identifier>gs:NetcdfOutput</ows:Identifier>
```

Comment:
I don't think we need this line. We have an Identifier on line 16 (loading dynamically from the `jobType` variable)